apps/src/applab/applab.js

```diff
@@ -8,14 +8,14 @@
 'use strict';
 
 var studioApp = require('../StudioApp').singleton;
-var commonMsg = require('../../locale/current/common');
-var applabMsg = require('../../locale/current/applab');
+var commonMsg = window.blockly.locale;
+var applabMsg = window.blockly.applab_locale;
```

Comment:
Moving away from a require and instead depending on a global being defined here seems backwards. Can we not use the require approach?